src/cpufreq.c

```diff
@@ -28,18 +31,52 @@
 
 static int num_cpu = 0;
 
+static char const * freq_fname = "/sys/devices/system/cpu/cpu%d/cpufreq/"
+								 "scaling_cur_freq";
+static char const * freq_fname_rhel = "/sys/devices/system/cpu/cpu%d/cpufreq/"
```

Comment:
Ack. "_alt" or "_old" and "_new" are fine by me if we don't have more details. I'd add a comment like:

    /* The path was renamed at some point.
     * The new path was first seen in CentOS 7.1. */

That'll explain what's going on and will still allow people to dig into it later. Sounds good?